Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -457,10 +457,11 @@ public extension UIView {
 		// https://videos.letsbuildthatapp.com/
 		translatesAutoresizingMaskIntoConstraints = false
 		if let superview = superview {
-			leftAnchor.constraint(equalTo: superview.leftAnchor).isActive = true
-			rightAnchor.constraint(equalTo: superview.rightAnchor).isActive = true
-			topAnchor.constraint(equalTo: superview.topAnchor).isActive = true
-			bottomAnchor.constraint(equalTo: superview.bottomAnchor).isActive = true
+			let left = leftAnchor.constraint(equalTo: superview.leftAnchor)
```

Comment:
Hey @viktart :)) 
Can you explain how this implementation is more performant that the previous? 
I believe the code is more clear this way, but can't see the performance gain here

Reply:
The performance gain is because you're not supposed to activate constraints individually. It's better to activate them all at once using the `NSLayoutConstraint.activate` API